src/Exporters/LinuxExporter.ts

```diff
@@ -305,4 +307,95 @@ export class LinuxExporter extends Exporter {
 		this.p('</CodeBlocks_project_file>');
 		this.closeFile();
 	}
+
+	exportCompileCommands(project: Project, _from: string, to: string, platform: string, vrApi: any, options: any) {
+		let from = path.resolve(process.cwd(), _from);
+		// TODO : assembly files, precompiled headers and all that stuff
+		// compile_commands.json is primarily for code completion so those things shouldn't matter too much
+		this.writeFile(path.resolve(to, 'compile_commands.json'));
+		let includes = [];
+		for (let inc of project.getIncludeDirs()) {
+			includes.push('-I');
+			includes.push(path.resolve(from, inc));
+		}
+		let defines = [];
+		for (let def of project.getDefines()) {
+			defines.push('-D');
+			defines.push(def.value.replace(/\"/g, '\\"'));
+		}
+		let libs = [];
+		for (let lib of project.getLibs()) {
+			libs.push('-l' + lib);
+		}
+		let optimization = options.debug ? '-O2' : '-g';
```

Comment:
should be `!options.debug`?

Reply:
Or `options.debug ? '-g' : '-O2'`. Anyway, it's fixed now.